GitUp/Application/Info.plist

```diff
@@ -33,6 +33,19 @@
 	<string>${BUNDLE_VERSION}</string>
 	<key>CFBundleShortVersionString</key>
 	<string>${BUNDLE_VERSION_STRING}</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
```

Comment:
I mean `CFBundleURLName`: just remove it since it's not required and GitHub Mac doesn't have it either.